client/src/components/Header/Header.tsx

```diff
@@ -6,13 +6,13 @@ import { useTheme } from "@material-ui/core/styles";
 function Header({
   title,
   subtitle,
-  debug,
   button,
+  payload,
 }: {
   title: React.ReactElement;
   subtitle?: React.ReactElement;
-  debug?: React.ReactElement;
   button?: React.ReactElement;
+  payload?: React.ReactElement;
```

Comment:
what does this payload contain?

Reply:
optional element to display under the Title (for UserHeader was used to display time, CartHeader used to display toggle switches)

Reply:
can name this something better? childComponent is also fine. Payload seems to be overloaded

Reply:
ok, maybe a neutral `content` then?